source/user-manual/ruleset/mitre.rst

```diff
@@ -203,78 +203,91 @@ Click on the **JSON** tab to view the details of the alert in JSON format:
+         "signed": "true",
+         "signatureStatus": "Valid",
+         "processGuid": "{ee63de90-ecf3-663a-c302-000000000a00}",
+         "processId": "3464",
+         "utcTime": "2024-05-08 03:09:39.440",
+         "hashes": "SHA1=246D0046CE546C86A3697129DE0DA7AC0F879273,MD5=64B5FECB5CE8C2B2D45D8C106FF185D3,SHA256=9365D1B5F8C5E94B8F1241AA2B18F72DBB08B3B4E5610ADF4FC6BFF4A5B0DAB4,IMPHASH=D7BA1B7058D2D7CD746B2F1CE3A81AB4",
+         "ruleName": "technique_id=T1053,technique_name=Scheduled Task",
+         "company": "Microsoft Corporation",
+         "fileVersion": "10.0.22621.1 (WinBuild.160101.0800)",
+         "user": "NT AUTHORITY\\\\SYSTEM"
+       },
+       "system": {
+         "eventID": "7",
+         "keywords": "0x8000000000000000",
+         "providerGuid": "{5770385f-c22a-43e0-bf4c-06f5698ffbd9}",
+         "level": "4",
+         "channel": "Microsoft-Windows-Sysmon/Operational",
+         "opcode": "0",
+         "message": "\"Image loaded:\r\nRuleName: technique_id=T1053,technique_name=Scheduled Task\r\nUtcTime: 2024-05-08 03:09:39.440\r\nProcessGuid: {ee63de90-ecf3-663a-c302-000000000a00}\r\nProcessId: 3464\r\nImage: C:\\Windows\\UUS\\Packages\\Preview\\amd64\\MoUsoCoreWorker.exe\r\nImageLoaded: C:\\Windows\\System32\\taskschd.dll\r\nFileVersion: 10.0.22621.1 (WinBuild.160101.0800)\r\nDescription: Task Scheduler COM API\r\nProduct: Microsoft® Windows® Operating System\r\nCompany: Microsoft Corporation\r\nOriginalFileName: taskschd.dll\r\nHashes: SHA1=246D0046CE546C86A3697129DE0DA7AC0F879273,MD5=64B5FECB5CE8C2B2D45D8C106FF185D3,SHA256=9365D1B5F8C5E94B8F1241AA2B18F72DBB08B3B4E5610ADF4FC6BFF4A5B0DAB4,IMPHASH=D7BA1B7058D2D7CD746B2F1CE3A81AB4\r\nSigned: true\r\nSignature: Microsoft Windows\r\nSignatureStatus: Valid\r\nUser: NT AUTHORITY\\SYSTEM\"",
+         "version": "3",
+         "systemTime": "2024-05-08T03:09:39.4557590Z",
+         "eventRecordID": "11838",
+         "threadID": "4816",
+         "computer": "DESKTOP-UV1HVEI",
+         "task": "7",
+         "processID": "4004",
+         "severityValue": "INFORMATION",
+         "providerName": "Microsoft-Windows-Sysmon"
+       }
+     }
+   },
+   "rule": {
+     "firedtimes": 2,
+     "mail": false,
+     "level": 4,
+     "description": "Process loaded taskschd.dll module. May be used to create delayed malware execution",
+     "groups": [
+       "sysmon",
+       "sysmon_eid7_detections",
+       "windows"
+     ],
+     "mitre": {
+       "technique": [
+         "Scheduled Task"
+       ],
+       "id": [
+         "T1053.005"
+       ],
+       "tactic": [
+         "Execution",
+         "Persistence",
+         "Privilege Escalation"
+       ]
+     },
+     "id": "92154"
+   },
+   "decoder": {
+     "name": "windows_eventchannel"
+   },
+   "input": {
+     "type": "log"
+   },
+   "@timestamp": "2024-05-08T03:09:40.518Z",
+   "location": "EventChannel",
+   "id": "1715137780.2747372",
+   "timestamp": "2024-05-08T03:09:40.518+0000",
```

Comment:
Needs to be one-space to the right

Reply:
Solved in https://github.com/wazuh/wazuh-documentation/pull/7280/commits/12f046a5a48400969961bd7d801b2db57f4bcbfe